ckanext/resource_visibility/helpers.py

```diff
@@ -0,0 +1,82 @@
+# encoding: utf-8
+
+import logging
+
+import ckan.plugins.toolkit as tk
+import ckan.model as model
+
+from ckanext.data_qld import (
```

Comment:
This plugin shouldn't depend on ckanext-data-qld. It should either have its own copy of those functions, or just be part of ckanext-data-qld.